cloudsync/smartsync.py

```diff
@@ -427,3 +427,15 @@ def smart_delete_path(self, local_oid, local_path):
                 self.state.update_entry(ent, LOCAL, local_oid, path=local_path, changed=True, exists=False)
                 self.state.requestset.add(ent)
                 self.state.excludeset.discard(ent)
+
+    def smart_rename(self, side, oid, new_path) -> str:
+        """smartsync aware rename"""
+        # oid MUST exist on the specified side
+        #   then, if target exists on same side, renaming on that side will fail in the rename
+        #   if target exists only on the other side, then there will be a conflict
```

Comment:
I'm a little confused on this comment. If the target exists on the other side won't we raise an error?

Reply:
The comment reflected an earlier state of the code -- I fixed the comment